pxr/usd/sdf/pathParser.h

```diff
@@ -0,0 +1,182 @@
+//
+// Copyright 2016 Pixar
+//
+// Licensed under the Apache License, Version 2.0 (the "Apache License")
+// with the following modification; you may not use this file except in
+// compliance with the Apache License and the following modification to it:
+// Section 6. Trademarks. is deleted and replaced with:
+//
+// 6. Trademarks. This License does not grant permission to use the trade
+//    names, trademarks, service marks, or product names of the Licensor
+//    and its affiliates, except as required to comply with Section 4(c) of
+//    the License and to reproduce the content of the NOTICE file.
+//
+// You may obtain a copy of the Apache License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the Apache License with the above modification is
+// distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the Apache License for the specific
+// language governing permissions and limitations under the Apache License.
+//
+#ifndef PXR_USD_SDF_PATH_PARSER_H
+#define PXR_USD_SDF_PATH_PARSER_H
+
+#include "pxr/pxr.h"
+#include "pxr/base/tf/stringUtils.h"
+#include "pxr/usd/sdf/path.h"
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+bool
+Sdf_ParsePath(std::string const &pathStr, SdfPath *path, std::string *errMsg);
+
+namespace Sdf_PathParser {
+
+////////////////////////////////////////////////////////////////////////
+// SdfPath grammar:
+
```

Comment:
With this being in a header, I'm glad to see the use of the explicit namespace of tao::TAO_PEGTL_NAMESPACE.
But it's a bit verbose and hard to read. I wondering if a `using PEGTL = tao::TAO_PEGTL_NAMESPACE;` here (or PEGTL_NS) would give us the best of both worlds yielding a shorter but still explicit namespace for all the declarations.

Reply:
shortend namespace with `namespace PEGTL_NS = tao::TAO_PEGTL_NAMESPACE;`